test/suite/test_cursor18.py

```diff
@@ -309,7 +309,7 @@ def test_prepare(self):
         version_cursor.set_key(1)
         self.assertEquals(version_cursor.search(), 0)
         self.assertEquals(version_cursor.get_key(), 1)
-        self.verify_value(version_cursor, 1, 0, WT_TS_MAX, WT_TS_MAX, 3, 1, 8, 0, 0)
```

Comment:
Do you understand why the flags changed here?

Reply:
Yes. The `WT_UPDATE_FIXED_HS` flag is removed as part of this change. Due to this, the flag numbers are modified.